src/plot_recipes.jl

```diff
@@ -227,39 +163,43 @@ end
 # ==============================
 
 """
-    plot_polygon(P::AbstractPolytope; ...)
+    plot_polytope(P::AbstractPolytope, [ε]::N=zero(N); ...) where {N<:Real}
 
 Plot a 2D polytope as the convex hull of its vertices.
 
 ### Input
 
 - `P` -- polygon or polytope
+- `ε` -- (optional, default: `0.0`) ignored, used for dispatch
```

Comment:
```suggestion
- `ε` -- (optional, default: `0`) ignored, used for dispatch
```
several times in this file